content/docs/command-reference/list-url.md

```diff
@@ -0,0 +1,97 @@
+# list-url
+
+> Aliased to `dvc ls-url`
+
+List contents from a supported URL (for example `s3://`, `ssh://`, and other
+protocols).
+
+> Useful to find data to `dvc get-url` or `dvc import-url`.
+
+## Synopsis
+
+```usage
+usage: dvc ls-url [-h] [-q | -v] [-R] url
+
+positional arguments:
+  url              See `dvc get-url` for full list of supported URLs.
```

Comment:
```suggestion
usage: dvc list-url [-h] [-q | -v] [-R] url

positional arguments:
  url              See `dvc list-url` for full list of supported URLs.
```
☝🏼 Seems copy/pasted from `get-url` ref. Is this the actual output of `list-url -h`?